packages/mds-agency/tests/sandbox-admin-request-handlers.ts

```diff
@@ -73,11 +73,11 @@ describe('Sandbox admin request handlers', () => {
       Sinon.replace(cache, 'wipeDevice', Sinon.fake.resolves(1))
       Sinon.replace(db, 'wipeDevice', Sinon.fake.resolves('it-worked'))
       await wipeDevice(
-        {
+        ({
           params: { device_id },
           query: { cached: false },
           get: Sinon.fake.returns('foo') as any
-        } as AgencyApiRequest<{ device_id: string }>,
+        } as unknown) as AgencyApiRequest<{ device_id: string }>,
```

Comment:
Why is this extra cast to `unknown` necessary?

Reply:
@janedotx In the case that types don't align at all during a typecast, Typescript will complain that you're _really_ doing something stupid with your direct cast. The cast to `unknown` circumvents that.

Reply:
Ah yeah sinon mocks can get funky since we know we only need to specify certain fields but the compiler isn't in on the jankiness.